exercises/01_sort4/sort4.go

```diff
@@ -20,8 +23,6 @@ func main() {
 	fmt.Print("Enter 4 numbers: ")
 	var a, b, c, d int
 	fmt.Scan(&a, &b, &c, &d)
-
```

Comment:
nit: I suggested to remove only one empty line. It's useful to have empty lines that separate logical code blocks.